app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,63 @@
+class InvitationsController < ApplicationController
+
+	def new
+		@invitation = Invitation.new
+	end
+	
+	def create
+		@invitation = Invitation.new(invitation_params)
+		@invitation.sender = current_user
+		@invitation.accepted = false
+		if @invitation.save
+			flash[:notice] = "Thank you, invitation created."
+			InvitationMailer.confirmation_email(@invitation).deliver
+			redirect_to users_path
+		else
+	    	flash[:alert] = "something is wrong :/"
+	    	redirect_to users_path
```

Comment:
After failed creation or update we should stay on new/edit page - that will allow us for example to highlight bad fields and show errors